src/test/resources/type-without-key.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": {
+    "en": "typeName"
+  },
+  "description": {
+    "en": "description"
+  },
+  "resourceTypeIds": [
+    "category"
+  ],
+  "fieldDefinitions": [
```

Comment:
One field is enough.

Reply:
created a new PR and please check this commit for the fix. https://github.com/commercetools/commercetools-project-sync/pull/213/commits/8030e3bdb0a9252deeb2e8a2b519333cc58d0861